Fw/Buffer/Buffer.hpp

```diff
@@ -73,7 +73,7 @@ class Buffer : public Fw::Serializable {
     //! \param data: data pointer to wrap
     //! \param size: size of data located at data pointer
     //! \param context: user-specified context to track creation. Default: no context
-    Buffer(U8* data, SizeType size, U32 context = NO_CONTEXT);
+    Buffer(U8* data, FwSizeType size, U32 context = NO_CONTEXT);
```

Comment:
I meant everything should use 'SizeType' instead as that would be consistent with all uses internal and external to the file.

If we are going this route (fine) but we should consider deleting SizeType herein....which would cause many problems.